app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<%= render "form", transaction: @transaction %>
```

Comment:
Please make sure the `Add Transaction` button sticks to the bottom of the page as it can be seen in the design.
![add-trans](https://user-images.githubusercontent.com/35504149/164572963-91111986-5bb3-4c5d-9961-b4af5aca2f84.png)

Reply:
Thank you for that. I fixed it